### ISSUE TITLE
[HW] Char Ryder

### ISSUE THREAD

**Issue by bl-kt** [datastaxdevs/workshop-graphql-netflix#450]
**Name:** Char Ryder
**Email:** char.ryder@blkt.co.uk
**Linkedin Profile:** linkedin.com/in/blkt

Attach the homework screenshots below:
-----------------------------------------

<SCREENSHOTS>

![Netflix Step 3](https://user-images.githubusercontent.com/56833452/126992182-225624b2-2477-48e6-8276-083cfc6d1022.png)
![Netflix Step 4](https://user-images.githubusercontent.com/56833452/126992186-baa41d66-aef0-41a8-b05f-6ffc21c773b8.png)
![Netflix Step 5](https://user-images.githubusercontent.com/56833452/126992190-ccf893d4-b353-4b60-abae-953831658604.png)
![Netflix Step 7](https://user-images.githubusercontent.com/56833452/126992193-4fa413b8-751f-4bf8-bca6-a858babdee26.png)
![Netflix Step 8b](https://user-images.githubusercontent.com/56833452/126992197-7db47fd6-ad42-4b20-b502-76084054ecd8.png)
![netflix final](https://user-images.githubusercontent.com/56833452/126992199-bbff3ed4-63ed-4e31-b6ad-51f9eaadd29b.png)
![Netflix Step 8](https://user-images.githubusercontent.com/56833452/126992202-a4b0d8ea-9ba7-4731-9f1c-2e3876fc39a7.png)

**Comment by hemidactylus** [comment]
Hi, Char Ryder!
Thank you for the submission, you deployed to Netlify all right!

But I don't see the movie you added in your deployed application: could you please make sure you did and provide more information (screenshot, a link to Netlify + name of the movie) for us to check?

Thank you!

**Comment by bl-kt** [comment]
Hi there,

What step was it detailed in to add your own movie? I don't recall seeing it.

Thanks :)

**Comment by bl-kt** [comment]
Fixed. I feel it should've been more clearly displayed in the actual instructions that an additional movie was to be added. I added Spirit: Stallion of the Cimarron to Children & Family. Here's a screenshot, the GIF didn't work and I could not find any MP4s. Here's my netlify link too.

![image](https://user-images.githubusercontent.com/56833452/127003940-dc733ac6-38ab-4b13-86f5-e090b0148761.png)
https://app.netlify.com/sites/suspicious-mirzakhani-595ab5/overview
